src/core-services/product/schemas/product.graphql

```diff
@@ -19,7 +19,10 @@ type Product {
   isVisible: Boolean!
 
   "All media for a product"
-  media: [ImageInfo]
+  media(
+    "Determines whether variant media should be included in the product or not"
+    shouldIncludeVariantMedia: Boolean
```

Comment:
You can do `shouldIncludeVariantMedia: Boolean = true` here, too, so that it's clear in the schema explorer that the default is true.

Reply:
updated